docs/WPML-url_to_postid.trace.MD

```diff
@@ -0,0 +1,33 @@
+```
```

Comment:
I'll probably remove this later but it's nice to put line comments on this here.